app/src/renderer/components/common/TableInvoice.vue

```diff
@@ -50,15 +38,15 @@ export default {
   },
   data: () => ({
     full,
+    viewDenom,
     info: `Estimated network fees based on simulation.`
   }),
   computed: {
-    ...mapGetters([`session`]),
+    ...mapGetters([`bondDenom`]),
     estimatedFee() {
       return (
         Number(this.gasPrice) *
-        Number(this.gasEstimate) *
-        this.session.gasAdjustment
```

Comment:
how so? why is the gas adjustment needed? 

in conversation with zaki yesterday i came to understand that the simulation provides an estimate of how much gas will be required - and then we multiply that by the gas price - which should be sufficient. the network is still accepting 0 fee txs - multiplying this by 1.5 (the adjustment) doesn't seem necessary to me.

Reply:
Agree, but when I implemented the fees I got the `insufficient fees` error multiple times, because the estimation was slightly inaccurate (for few `uatoms`)

Reply:
idk if it has changed I haven't try it out. But that was my experience back then